getResponse.js

```diff
@@ -1,69 +1,88 @@
-const axios = require('axios');
 
-function getResponse(messageText, data) {
+// splitIntoWords takes text and splits it by non-alphanumeric
+// characters (e.g. spaces, tabs, punctuation), into an array
+function splitIntoWords(text) {
+  return text.toLowerCase().split(/[^a-z0-9]/).filter(e => e);
+}
+
+// isWithinOperatingHours takes two strings representing opening
+// & closing times, each formatted as `hh:mm`, and returns true
+// if the current time is between those, false otherwise.
+function isWithinOperatingHours(openTime, closeTime) {
+  const currentDate = new Date();
+
+  // HACK: shifting current hours by 4 to account for time difference
+  const [currentHour, currentMinute] = [currentDate().getHours() - 4, currentDate.getMinutes()];
```

Comment:
@adelq Yeah it doesn't, probably the best solution will be to use momentjs to handle the dates